locales/en-US/notes.properties

```diff
@@ -48,9 +48,11 @@ emptyPlaceHolder=Take a note…
 giveFeedback=Click here to give us some feedback
 feedback=Give Feedback
 cancelLogin=Cancel Login
+cancelSetup=Cancel Setup
```

Comment:
What's "Setup" in this context?

Reply:
I have a screenshot (to keep working if this kind of thing happen), but might be deprecated at any time.
<img width="1478" alt="capture d ecran 2018-03-08 a 07 41 57" src="https://user-images.githubusercontent.com/3018618/37255016-cbfe3dfc-2578-11e8-82e2-c84a8f04ad71.png">

Reply:
Thanks. So, I guess "Setup" stands for login+verification.

Reply:
I feel like that's the prefect way to say it, YES! 😍